libs/schematic/api-description/src/paths/nodes/@{nodeDisplay}/validationRules.yaml

```diff
@@ -0,0 +1,19 @@
+parameters:
+  - $ref: ../../../components/parameters/path/nodeDisplay.yaml
+get:
+  tags:
+    - Schema
+  summary: Gets the validation rules associated with a given node
```

Comment:
Here again, we should also specify that the rule arguments are returned as well if that's the case